### PR TITLE
Milestone 1 : Initialize project with components (React only).

### PR DESCRIPTION
**Pull request by Chrissiku**
### What I use 👨‍💻  
- [x] React Router.
- [x]  React Components

### What I implement ♻️ 
- Initialise React app.
- Set as re-usable components
- Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two `<Route>`s and `<Link>`s for the app's navigation
- 

## Technology used 🧑‍💻 
- Language : JavaScript(ES6), JSX and CSS3
- Framework: None
- Library: `Reactjs` 
- Module bundler : `Babel`
- Technology: Gitflow, Github Workflow, Linters

## Project New Overview
book

### CODE REVIEW

File: src/components/homePage/Book.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          <li>
+            <a>Edit</a>
+          </li>
+        </ul>
+      </div>
+
+      <div className="right-book">
+        <div className="progress">
+          <div className="circle">
+            <div className="inner"></div>
```
Comment (aimemalaika) [inline src/components/homePage/Book.js]: - [x] Kindly make sure that the book component has the two required props `title` and `author`

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @Chrissiku, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Chrissiku 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chrissiku** [comment]
Thank you @OmarRamoun for the approval of the project

**Pull request merged**